modules/mig_with_percent/main.tf

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  distribution_policy_zones = {
+    default = ["${data.google_compute_zones.available.names}"]
+    user    = ["${var.distribution_policy_zones}"]
+  }
+}
+
+data "google_compute_zones" "available" {}
```

Comment:
This should be configured with `region = "${var.region}"`.

Reply:
Ack, thanks for pointing this out.
[This](https://github.com/terraform-google-modules/terraform-google-vm/blob/1e5c4b229251388f7fa2e160a29c30cdc367f627/modules/mig/main.tf#L24) should also be updated, but I'll do that after merging this series.